lib/beetle/client.rb

```diff
@@ -55,15 +55,15 @@ def initialize(config = Beetle.config)
       @messages = {}
       @bindings = {}
       @deduplication_store = DeduplicationStore.new(config)
-      @dead_lettering = DeadLettering.new(config)
+      @queue_properties = QueueProperties.new(config)
       load_brokers_from_config
       register_exchange(config.beetle_policy_exchange_name)
       # make sure dead lettering is false for the policy update queue
       register_queue(
         config.beetle_policy_updates_queue_name,
         :exchange => config.beetle_policy_exchange_name,
         :key => config.beetle_policy_updates_routing_key,
-        :dead_lettering => false,
+        :queue_properties => false,
```

Comment:
The comment above still says *make sure dead lettering is false for the policy update queue*, but *dead lettering* does not appear anymore.

To be honest, this interface is not very intuitive. `:queue_properties` sounds like an options hash to me, not like boolean as used here. And then why have an extra `:lazy` option. Shouldn't this be part of `:queue_properties`?

Reply:
renamed too many instances of dead_lettering with search and replace. fixed.